internal/commands/profiling/asyncprofiler/asyncprofiler.go

```diff
@@ -0,0 +1,17 @@
+package asyncprofiler
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+var Command = &cli.Command{
+	Name:      "asyncprofiler",
```

Comment:
If this command is under the `profiling` command, should we remove the `profiler` suffix? 
For now, the command should be: `swctl profiling asyncprofiler`, I think it could be: `swctl profiling async` is much clear, what do you think?